packages/api/AlvTimeWebApi/ErrorHandling/ErrorHandlingMiddleware.cs

```diff
@@ -23,6 +24,24 @@ public async Task Invoke(HttpContext context)
             {
                 await _next(context);
             }
+            catch (ValidationException ve)
+            {
+                context.Response.StatusCode = 400;
+                context.Response.ContentType = "application/json";
+
+                var problem = new ProblemDetails
+                {
+                    Status = context.Response.StatusCode,

```

Comment:
Also, we should maybe create some integration tests to assert the http status code, in case fronted rely in the given code. Could be a seperate github issue